js/ace/mode-verilog.js

```diff
@@ -5,7 +5,7 @@ var oop = require("../lib/oop");
 var TextHighlightRules = require("./text_highlight_rules").TextHighlightRules;
 
 var VerilogHighlightRules = function() {
-var keywords = "always|and|assign|automatic|begin|buf|bufif0|bufif1|case|casex|casez|cell|cmos|config|" +
+var keywords = "always|always_comb|always_ff|and|assign|automatic|begin|buf|bufif0|bufif1|case|casex|casez|cell|cmos|config|" +
```

Comment:
Unfortunately, I can't accept changes to the Ace files. If you want these in, you'll need to open a pull request upstream to the main Ace repo, and then I'll pull them in when they release.